tests/gla/block_mat_02.with_trilinos=true.mpirun=1.output.2

```diff
@@ -0,0 +1,769 @@
+
+DEAL:0:PETSc::numproc=1
+{[0,40]}
+Block (0, 0)
+(0,0) (0.00000,0.00000)
+(0,1) (0.00000,0.00000)
```

Comment:
i would say so. That's exactly the output of the real-valued part `gla/block_mat_02.with_trilinos=true.mpirun=1.output`.